cluster/network.go

```diff
@@ -284,9 +339,88 @@ func (c *Cluster) doWeaveDeploy(ctx context.Context, data map[string]interface{}
 	return c.doAddonDeploy(ctx, pluginYaml, NetworkPluginResourceName, true)
 }
 
+func (c *Cluster) doAciDeploy(ctx context.Context, data map[string]interface{}) error {
+	refreshTime, err := atoiWithDefault(c.Network.AciNetworkProvider.ApicRefreshTime, DefaultApicRefreshTime)
+	if err != nil {
+		return err
+	}
+	serviceMonitorInterval, err := atoiWithDefault(c.Network.AciNetworkProvider.ServiceMonitorInterval, DefaultServiceMonitorInterval)
+	if err != nil {
+		return err
+	}
+	_, clusterCIDR, _ := net.ParseCIDR(c.ClusterCIDR)
```

Comment:
How are input errors handled here? Errors are ignored and there is no validation in `cluster/validation` added

Reply:
Thanks, added the validation back for the ignored cases. Cluster/validation seems to be for common(to all CNI) network options, so not adding there.